src/runtime/tailwind.config.cjs

```diff
@@ -0,0 +1,35 @@
+const { resolve } = require('path')
+const { defaultExtractor: createDefaultExtractor } = require('tailwindcss/lib/lib/defaultExtractor.js')
+
+const defaultExtractor = createDefaultExtractor({ tailwindConfig: { separator: ':' } })
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  darkMode: 'class',
+  plugins: [
+    require('@tailwindcss/forms')({ strategy: 'class' }),
+    require('@tailwindcss/aspect-ratio'),
+    require('@tailwindcss/typography'),
+    require('@tailwindcss/container-queries'),
+    require('@headlessui/tailwindcss')
+  ],
+  content: {
+    files: [
+      resolve('<%= options.runtimeDir %>', 'components/**/*.{vue,mjs,ts}'),
+      resolve('<%= options.runtimeDir %>', 'ui.config/**/*.{mjs,js,ts}')
+    ],
+    transform: {
+      vue: (content) => {
+        return content.replaceAll(/(?:\r\n|\r|\n)/g, ' ')
+      }
+    },
+    extract: {
+      vue: (content) => {
+        return [
+          ...defaultExtractor(content),
+          // ...customSafelistExtractor(options.prefix, content, nuxt.options.appConfig.ui.colors, options.safelistColors)
```

Comment:
Pending migration

Reply:
@benjamincanac any help in moving `colors.ts` logic would be appreciated! 🙂

Reply:
I think we should be able to simply move it to `runtime/` (and import it in `module.ts` as needed); this is something that is being done in the tailwind module for `configMerger` util - https://github.com/nuxt-modules/tailwindcss/pull/795/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R74